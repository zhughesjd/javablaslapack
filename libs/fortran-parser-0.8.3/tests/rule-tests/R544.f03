
! Testing saved entity list
save this, that, theother
end
