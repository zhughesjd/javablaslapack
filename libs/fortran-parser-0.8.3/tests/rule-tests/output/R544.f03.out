1101:begin
544:list-begin
544:list 3
1103
1101 false false false
